pypdf/_writer.py

```diff
@@ -129,9 +124,8 @@
 
 logger = logging.getLogger(__name__)
 
-
 OPTIONAL_READ_WRITE_FIELD = FieldFlag(0)
-ALL_DOCUMENT_PERMISSIONS = UserAccessPermissions((2**31 - 1) - 3)
+ALL_DOCUMENT_PERMISSIONS = UserAccessPermissions((2 ** 31 - 1) - 3)
```

Comment:
```suggestion
ALL_DOCUMENT_PERMISSIONS = UserAccessPermissions((2**31 - 1) - 3)
```

Reply:
This is caused by an old version of black. Interesting, that CI didn't complain :thinking:

Reply:
this was made by `ruff . --fix`